src/search.rs

```diff
@@ -17,7 +18,7 @@ pub struct SearchResult<T> {
     pub result: T,
     /// The formatted result.
     #[serde(rename = "_formatted")]
-    pub formatted_result: Option<T>,
+    pub formatted_result: Option<Map<String, Value>>,
```

Comment:
It is a simplification of types due to the fact that number fields are converted to strings?

Reply:
Yes!